appcues/src/main/java/com/appcues/data/model/Experience.kt

```diff
@@ -27,6 +27,7 @@ internal data class Experience(
     val requestId: UUID? = null,
     val error: String? = null,
     var renderErrorId: UUID? = null,
+    var previewQuery: Map<String, String> = mapOf(),
```

Comment:
I feel like this keeps it all more organized, so when on ViewPresenter we want to call refreshPreview we can just grab both the ID and previewQuery from the experience.